system/include/emscripten/fetch.h

```diff
@@ -93,8 +93,10 @@ typedef struct emscripten_fetch_attr_t
 
 	// If non-zero, specifies a pointer to the data that is to be passed as the body (payload) of the request
 	// that is being performed. Leave as zero if no request body needs to be sent.
-	// The memory pointed to by this field is provided by the user, and needs to be valid only until the call to
-	// emscripten_fetch() returns.
+    // The memory pointed to by this field is provided by the user, and needs to be valid throughout the
```

Comment:
Looks like there might be spaces mixed with tabs, given the indentation looks wrong? Please use only spaces.

Reply:
Yeah sorry not sure why that is, should all be spaces, I'll have a check through

Reply:
I think the actual issue is I'm using spaces but the original file is tabs.

Reply:
Can we change then this PR to use also tabs to stay with the existing style of surrounding code?

(And then adhering to spaces convention could be done in a separate "style change only" PR, because such changes wreak havoc on the changed lines in the PR, and also on Git Blame history. So such changes would be good to be done in a commit/PR of its own.)